samples/bluetooth/throughput/child_image/hci_rpmsg.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_BT_CTLR_TX_BUFFER_SIZE=251
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+CONFIG_BT_RX_BUF_LEN=255
+
+CONFIG_BT_MAX_CONN=4
```

Comment:
Does this sample support more than one connection?

Reply:
Yes. I reduced it to minimal 2 connections.